packages/editor/src/components/parts/program-start/activity/StartPart.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { DeepPartial, render, renderHook, screen } from 'test-utils';
+import { ElementData, InscriptionValidation, ProgramInterfaceStartData } from '@axonivy/inscription-protocol';
+import { PartStateFlag } from '../../../editors';
+import { useProgramInterfaceStartPart } from './StartPart';
+
+const Part = () => {
+  const part = useProgramInterfaceStartPart();
+  return <>{part.content}</>;
+};
+
+describe('StartPart', () => {
+  function renderPart(data?: ProgramInterfaceStartData) {
+    render(<Part />, {
+      wrapperProps: { data: data && { config: data } }
+    });
+  }
+
+  test('empty data', async () => {
+    renderPart();
+    expect(screen.queryByText('Program')).toBeInTheDocument();
+    expect(screen.queryByText('Timeout')).toBeInTheDocument();
+  });
+
+  test('full data', async () => {
+    renderPart({
+      javaClass: 'Test',
+      exceptionHandler: '>> Ignore Exception',
+      timeout: {
+        seconds: '',
+        error: 'ivy:error:program:timeout',
+        action: 'NOTHING',
+        duration: ''
+      }
+    });
+    expect(screen.getByLabelText('Java Class')).toHaveValue('Test');
+  });
+
+  function assertState(expectedState: PartStateFlag, data?: DeepPartial<ProgramInterfaceStartData>, validation?: InscriptionValidation) {
+    const { result } = renderHook(() => useProgramInterfaceStartPart(), {
+      wrapperProps: { data: data && { config: data }, validations: validation && [validation] }
+    });
+    expect(result.current.state.state).toEqual(expectedState);
+  }
+
+  test('configured', async () => {
+    assertState('empty');
+    assertState('configured', {
+      javaClass: 'Bla'
+    });

```

Comment:
I would put this on the same line to save space `assertState('configured', { javaClass: 'Bla' });`

And test other values too. I mean you currently only check that if the javaClass changes the accordion state changes to configured. What about the timeout? It's very easy to quickly check that your compareData function in the startpart is correctly implemented by check at least one of the configs in "timeout" and "exceptionHandler".